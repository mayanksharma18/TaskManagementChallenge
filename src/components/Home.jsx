import React from 'react';
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';
import Board from './Board/Board';

const Home = () => {
  return (
    <div>
    
        <AppBar position="static">
          <Toolbar>
            <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
              Task Management App
            </Typography>
          </Toolbar>
        </AppBar>
        <Toolbar sx={{ justifyContent: 'center' }}>
          <Board />
        </Toolbar>
    </div>
  );
};

export default Home;
