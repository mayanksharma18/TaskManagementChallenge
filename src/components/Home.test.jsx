import React from "react";
import { render, screen } from "@testing-library/react";
import Home from "./Home";
jest.mock("./Board/Board", () => () => (
  <div data-testid="mock-board">Mocked Board</div>
));

describe("Home", () => {
  it("renders Board component", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
