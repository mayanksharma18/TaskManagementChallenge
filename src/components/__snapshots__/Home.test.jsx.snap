// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Board component 1`] = `
<div>
  <div>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
        >
          Task Management App
        </div>
      </div>
    </header>
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-y44dvl-MuiToolbar-root"
    >
      <div
        data-testid="mock-board"
      >
        Mocked Board
      </div>
    </div>
  </div>
</div>
`;
